src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -32,24 +32,24 @@
 abstract class DoctrineType extends AbstractType
 {
     /**
-     * @var ManagerRegistry
+     * @var DoctrineChoiceLoader[]
      */
-    protected $registry;
+    private static $choiceLoaders = array();
```

Comment:
Updated. There was some reason why I made it static, but I can't remember. We'll find out.